joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1866,28 +1872,13 @@ class AstCreator(filename: String, global: Global)
       val callNodes = methodIdentifierAsts.head.nodes.filter(node => node.isInstanceOf[NewCall])
```

Comment:
Rather use `collect`

Reply:
If you use `collect` then `callNodes` will already be type cast to a collection of `Call` and you don't have to do the type casting on line 1874